### PR TITLE
Fixes part of #68026: revise some errors about golint in some packages

### PR DESCRIPTION
**Pull request by Sajfer**
**What this PR does / why we need it**:

Fix golint errors in modules:
test/e2e_node
test/e2e_node/builder
test/e2e_node/environment
test/e2e_node/remote
test/e2e_node/runner/remote
test/e2e_node/services

**Which issue(s) this PR fixes** 
part of #68026

**Special notes for your reviewer**:
After I revised above errors, I run hack/update-bazel.sh and hack/update-gofmt.sh to make sure I fix any problems with gofmt or bazel.
@dims 

```release-note
NONE
```

### CODE REVIEW

**Comment by Sajfer** [comment]
I signed it

**Comment by idealhack** [comment]
@Sajfer Thanks for your contribution! It seems you haven't signed the CLA yet.

**Comment by idealhack** [comment]
It seems you have to run `./hack/update-gofmt.sh` and amend your commit :)

**Comment by Sajfer** [comment]
Im working on fixing the errors left, but one lint errors is "test/e2e_node/apparmor_test.go:17:1: don't use an underscore in package name", is there a preferred name to use instead? e2eNode? or should this error be skipped? @idealhack

**Comment by idealhack** [comment]
@Sajfer I think you'd like to fix it, since you removed `test/e2e_node` from `hack/.golint_failures`

File: test/e2e_node/doc.go [hunk 1]
```diff
@@ -14,6 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// e2e_node contains e2e tests specific to the node
+// Package e2e_node contains e2e tests specific to the node
 // TODO: rename this package e2e-node
```
Comment (idealhack) [inline test/e2e_node/doc.go]: you may try this one

**Comment by Sajfer** [comment]
I renamed the package to e2e-node, but I get the following error in apparmor_test.go: 
"Expected ';' but found '-', line: 17 col: 12". And the line it points to is "package e2e-node". So im not sure if it collides with the package called e2e

**Comment by idealhack** [comment]
@Sajfer Hi, do you have the bandwidth to rebase and push this PR forward?

**Comment by Sajfer** [comment]
I can update it, but I would need input on what to name the e2e_node module.

**Pull request closed without merging**